tests/test_examples.py

```diff
@@ -55,7 +61,7 @@ def test_examples(solver, example: Example):
     if example.skip_solvers is not None and solver in example.skip_solvers:
         pytest.skip(f"Skipping {solver} for example {example.folder_name}")
 
-    example_dir = Path("tests/examples") / example.folder_name
+    example_dir = Path("examples") / example.folder_name
```

Comment:
All code should be run from the root folder to ensure that the tests continue to work properly in other environments (e.g. CI pipeline)